doc/plutus-core-spec/untyped-grammar.tex

```diff
@@ -30,6 +30,7 @@ \subsection{Lexical grammar}
 \end{minipage}%
 \nomenclature[C]{$L,M,N$}{A term}%
 \nomenclature[C]{$n$}{A name}%
+\nomenclature[C]{$i$}{An integer}%
```

Comment:
I'm not quite sure what to say here. In the binary format it's definitely constrained in size, but in principle in the abstract syntax it could be arbitrary. I guess the spec should match the implementation, so I'll try and say something.

Reply:
I'm just going to add a note.